introduction.md

```diff
@@ -1,2 +1,4 @@
 Hello i'm @elchesco and this is a fork for the course, :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?